### ISSUE TITLE
Worker running with only 8GB of memory on machine with 30GB

### ISSUE THREAD

**Issue by abyrd** [conveyal/r5#304]
A worker is running on an r3.xlarge which has 30GB of memory. Yet the worker reports:
```
"memoryMax": 7964983296,
"memoryTotal": 7964983296,
"memoryFree": 1707777000,
```

So the worker only sees about 8 GB of memory.

**Comment by abyrd** [comment]
The user data startup script does not seem to use the memory figure that it calculates:
```
# Figure out how much memory to give the worker
# figure out how much memory to use
TOTAL_MEM=`grep MemTotal /proc/meminfo | sed s/[^0-9]//g`
# 2097152 kb is 2GB, leave that much for the OS
MEM=`echo $TOTAL_MEM - 2097152 | bc`

# Start the worker
# run in ec2-users home directory, in the subshell
{
    cd ~ec2-user
    sudo -u ec2-user java8 -jar r5.jar worker worker.conf >> $LOGFILE 2>&1

    # If the worker exits or doesnt start, wait a few minutes so that the CloudWatch log agent grabs
    # the logs
    sleep 120
    halt -p
} &
```

**Comment by abyrd** [comment]
Fixed in 98f733673aea3083319203d6d68b46925bca272c
